tests/e2e/v3_curl_test.go

```diff
@@ -381,6 +381,34 @@ func testV3CurlMaintenanceAlarmMissiongAlarm(cx ctlCtx) {
 	}
 }
 
+func TestV3CurlMaintenanceHash(t *testing.T) {
+	testCtl(t, testV3CurlMaintenanceHash, withCfg(*newConfigNoTLS()))
+}
+
+func testV3CurlMaintenanceHash(cx ctlCtx) {
+	if err := cURLPost(cx.epc, cURLReq{
+		endpoint: "/v3/maintenance/hash",
+		value:    "{}",
+		expected: `,"revision":"1","raft_term":"2"}`,
+	}); err != nil {
+		cx.t.Fatalf("failed post maintenance hash request (%s) (%v)", cx.apiPrefix, err)
+	}
+}
+
+func TestV3CurlMaintenanceHashKV(t *testing.T) {
+	testCtl(t, testV3CurlMaintenanceHashKV, withCfg(*newConfigNoTLS()))
+}
+
+func testV3CurlMaintenanceHashKV(cx ctlCtx) {
+	if err := cURLPost(cx.epc, cURLReq{
+		endpoint: "/v3/maintenance/hashkv",
+		value:    `{"revision": 1}`,
+		expected: `,"revision":"1","raft_term":"2"}`,
```

Comment:
Unfortunately, 3.5 doesn't support regular expression matching. At least, let's differentiate the response between `/v3/maintenance/hashkv` and `/v3/maintenance/hash`.
```suggestion
		expected: `,"compact_revision":`,
```